gitlab-cng-17.2.yaml

```diff
@@ -32,8 +32,8 @@ var-transforms:
 
 package:
   name: gitlab-cng-17.2
-  version: 17.2.0
-  epoch: 1
+  version: 17.2.1
```

Comment:
Are we sure the other packages tracked above weren't bumped in GitLab's variables manifest?

Reply:
https://gitlab.com/gitlab-org/build/CNG/-/commit/7b7006df209c05f72b0fb5968c0bfb619d049d56 according to here they were not?

Reply:
Awesome!